src/combineReducers.js

```diff
@@ -41,6 +41,6 @@ export const combineReducers = (
 
   return loop(
     hasChanged ? model : state,
-    Cmd.batch(cmds)
+    Cmd.list(cmds, {batch: true}) //todo: remove batch in 5.0
```

Comment:
don't want to change the behavior until a breaking change to avoid catching people off guard.